demos/functions/views/demo/workbox-background-sync/sw.hbs

```diff
@@ -0,0 +1,13 @@
+{{{WORKBOX_SW_IMPORT}}}
+
+const bgSyncPlugin = new workbox.backgroundSync.Plugin('myQueueName');
+
+workbox.routing.registerRoute(new workbox.routing.Route(
```

Comment:
Do you need to use workbox.routing.route?

I though you could just pass in a function and a strategy.